pandas/core/generic.py

```diff
@@ -7623,12 +7624,24 @@ def between_time(
             Initial time as a time filter limit.
         end_time : datetime.time or str
             End time as a time filter limit.
-        include_start : bool, default True
+        include_start : bool
```

Comment:
I'd keep `, default True` in, as that's the practical effect (and would be consistent with what pandas does in the `replace` docstring)

Reply:
@MarcoGorelli is this how we annotate the default when this is the deprecated arg? (if so agree @suyashgupta01 that we need to revert this change)

Reply:
I Don't know if there's a general rule, this is just what's done in Dataframe.str.replace for the regex argument, where the signature shows `=None` but the docstring shows the default value that `regex` is set to if it's None. I'll have a look at other methods to see what's done there

Reply:
Not sure what I should do here.

Reply:
`squeeze` in `groupby` also shows the default behaviour in the docstring

> Not sure what I should do here.

just revert the change in this line (put the `, default True` back)

Reply:
likewise for `include_end`

Reply:
making the change in the next push :)